src/auth/presentation/guards/user-ownership.guard.ts

```
@@ -0,0 +1,52 @@
+import { Injectable, CanActivate, ExecutionContext, ForbiddenException } from '@nestjs/common';
```

ESLint

[error] Replace `Injectable,·CanActivate,·ExecutionContext,·ForbiddenException` with `CanActivate,·ExecutionContext,·ForbiddenException,·Injectable`

(prettier/prettier)